templates/al2023/provisioners/install-worker.sh

```diff
@@ -148,24 +148,12 @@ fi
 ###############################################################################
 sudo dnf install -y runc-${RUNC_VERSION}
 if [[ "$INSTALL_CONTAINERD_FROM_S3" == "true" ]]; then
-  CONTAINERD_BINARIES=(
-    containerd
-    containerd-shim-runc-v2
-    ctr
-  )
-  for binary in "${CONTAINERD_BINARIES[@]}"; do
-    echo "Installing containerd from S3..."
-    aws s3 cp --region ${BINARY_BUCKET_REGION} s3://${BINARY_BUCKET_NAME}/containerd/${CONTAINERD_VERSION}/${MACHINE}/${binary} .
-    sudo chmod +x $binary
-    sudo mv $binary /usr/bin/
-    # exclude containerd from yum.conf as versionlock doesn't work in this case
-    echo "exclude=containerd*" | sudo tee -a /etc/dnf/dnf.conf
-  done
-  sudo mkdir -p /var/lib/containerd
+  aws s3 cp --region ${BINARY_BUCKET_REGION} s3://${BINARY_BUCKET_NAME}/containerd/${CONTAINERD_VERSION}/containerd-${CONTAINERD_VERSION}-1.eks.${MACHINE}.rpm /tmp/containerd/
+  sudo yum install -y /tmp/containerd/containerd-${CONTAINERD_VERSION}-1.eks.${MACHINE}.rpm
```

Comment:
nit:
```suggestion
  sudo dnf install -y /tmp/containerd/containerd-${CONTAINERD_VERSION}-1.eks.${MACHINE}.rpm
```
also the same comment about the `-1`

Reply:
Nice catch. Fixed. The `-1` is answered above